md/single-sign-on-with-saml.md

```diff
@@ -6,7 +6,7 @@
 
 This pages explains how to configure your Bonita Platform system to use the SAML protocol to provide single sign-on (SSO). It assumes you already have a SAML Identity Provider server up and running (IdP).
 
-This information applies to a Bonita platform deployed from a bundle (Tomcat or WildFly), not to the Engine launched from Bonita Studio. `<BUNDLE_HOME>` refers to the root directory of the bundle.
+This information applies to a Bonita platform deployed from a bundle, not to the Engine launched from Bonita Studio. `<BUNDLE_HOME>` refers to the root directory of the bundle.
```

Comment:
```suggestion
This information applies to a Bonita platform deployed from a Tomcat bundle, not to the Engine launched from Bonita Studio. `<TOMCAT_HOME>` refers to the root directory of the Tomcat bundle.
```

Reply:
Again we don't want Tomcat references when not needed